Error when passing APPCPATH to appclient.bat in windows

[//]: # "Please remove these comments"
[//]: # "Provide a general summary of the issue in the Title above"

## Environment Details
* GlassFish Version (and build number): Eclipse GlassFish  7.0.0  (build master-b551-g6f65b47 2022-05-30T04:52:27+0000)
* JDK version: openjdk version "11.0.9.1" 2020-11-04
* OS: windows 10
* Database: none

----------

## Problem Description
[//]: # "Describe the bug in detail highlighting current behavior vs expected behavior"
[//]: # "State if the problem is easily reproducible or happens intermittently"
[//]: # "Include stack traces or command outputs"
Parsing of parameters passed as APPCPATH values fails in windows.  
```cmd
>set APPCPATH=C:\work\file.jar & D:\work\glassfish7\glassfish\bin\appclient.bat -client C:\work\client.jar
java.lang.IllegalArgumentException: Could not parse the classpath: C:\work\file.jar
        at org.glassfish.appclient.common.ClassPathUtils.convertClassPathToPaths(ClassPathUtils.java:143)
        at org.glassfish.appclient.common.ClassPathUtils.convertClassPathToURLs(ClassPathUtils.java:124)
        at org.glassfish.appclient.common.ClassPathUtils.getJavaClassPathForAppClient(ClassPathUtils.java:87)
        at org.glassfish.appclient.client.CLIBootstrap$JVMMainOption.getClassPathForGfClient(CLIBootstrap.java:813)
        at org.glassfish.appclient.client.CLIBootstrap$JVMMainOption.processValue(CLIBootstrap.java:804)
        at org.glassfish.appclient.client.CLIBootstrap.run(CLIBootstrap.java:249)
        at org.glassfish.appclient.client.CLIBootstrap.main(CLIBootstrap.java:157)
Caused by: java.nio.file.InvalidPathException: Trailing char < > at index 16: C:\work\file.jar
        at java.base/sun.nio.fs.WindowsPathParser.normalize(WindowsPathParser.java:191)
        at java.base/sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:153)
        at java.base/sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:77)
        at java.base/sun.nio.fs.WindowsPath.parse(WindowsPath.java:92)
        at java.base/sun.nio.fs.WindowsFileSystem.getPath(WindowsFileSystem.java:229)
        at java.base/java.io.File.toPath(File.java:2292)
        at org.glassfish.appclient.common.ClassPathUtils.convertClassPathToPaths(ClassPathUtils.java:139)
        ... 6 more
```
This is due to the fact that in cmd.exe, when an environment variable is passed by connecting it with "&", if a trailing space is added, the space is also passed as the value of the environment variable.

This problem does not occur in GlassFish 5 and in linux.

## Steps to reproduce
[//]: # "Step by step instructions to reproduce the problem"
[//]: # "Provide sample code/application if relevant"
It is reproduced by the command line described above.   
(The application jar does not have to exist for this error.)

--- a/appserver/appclient/client/acc/src/main/java/org/glassfish/appclient/common/ClassPathUtils.java
+++ b/appserver/appclient/client/acc/src/main/java/org/glassfish/appclient/common/ClassPathUtils.java
@@ -136,7 +136,7 @@ public class ClassPathUtils {
         final List<Path> result = new ArrayList<>();
         try {
             for (String classPathElement : classPath.split(File.pathSeparator)) {
-                result.add(new File(classPathElement).toPath());
+                result.add(new File(classPathElement.trim()).toPath());
             }
             return result;
         } catch (Exception e) {

